### PR TITLE
feat: Add support for empty mounts in provisioning script

### PR DESCRIPTION
**Pull request by DerekMaggio**
# Overview

Add support for empty pipette mounts.

# Test Plan

- [x] Run configuration that has only a left pipette. Make sure that right pipette eeprom size is 0. Make sure left pipette eeprom size is not.

# Changelog

- Add logic to parse empty mount env var
- Add tests

# Review requests

None

# Risk assessment

Low

### CODE REVIEW

File: hardware/opentrons_hardware/scripts/emulation_pipette_provision.py [hunk 1]
```diff
@@ -20,36 +19,81 @@
 RIGHT_PIPETTE_ENV_VAR_NAME = "RIGHT_OT3_PIPETTE_DEFINITION"
 
 
-@dataclass
 class OT3PipetteEnvVar:
```
Comment (Laura-Danielle) [inline hardware/opentrons_hardware/scripts/emulation_pipette_provision.py:28]: Couldn't you just keep this a data class with defaults?

Comment (Laura-Danielle) [reply hardware/opentrons_hardware/scripts/emulation_pipette_provision.py:28]: or keep it a dataclass and use a `build` classmethod

Comment (DerekMaggio) [reply hardware/opentrons_hardware/scripts/emulation_pipette_provision.py:28]: Stop making so much sense. I have no idea why I did it that way. I will go back and fix it. 

If I remember the reason I did it that way, and the reason makes sense, I will drop a comment.

Comment (DerekMaggio) [reply hardware/opentrons_hardware/scripts/emulation_pipette_provision.py:28]: I remembered my reasoning for doing it that way. 
My reasoning was not good. 

Here is the fix
262d21e41748787d14a2b013f6c4d5401c7a6267

**Review by fsinapi — APPROVED** [review]
LGTM!

**Pull request merged**